File: auto-docs/executables/python/overlaid_histogram.py
import numpy as np
x0 = np.random.randn(500)
x1 = np.random.randn(500)+1
import plotly.plotly as py
from plotly.graph_objs import *
py.sign_in('TestBot', 'r1neazxo9w')
trace1 = Histogram(
    x=x0,
    opacity=0.75
)
trace2 = Histogram(
    x=x1,
    opacity=0.75
)
data = Data([trace1, trace2])
layout = Layout(
    barmode='overlay'
)
fig = Figure(data=data, layout=layout)
plot_url = py.plot(fig, auto_open=False, filename='overlaid-histogram')

File: auto-docs/executables/matlab/overlaid_histogram.m
import numpy as np
x0 = np.random.randn(500)
x1 = np.random.randn(500)+1
signin('TestBot', 'r1neazxo9w')
trace1 = struct(...
  'x', x0, ...
  'opacity', 0.75, ...
  'type', 'histogram');
trace2 = struct(...
  'x', x1, ...
  'opacity', 0.75, ...
  'type', 'histogram');
data = {trace1, trace2};
layout = struct('barmode', 'overlay');
response = plotly(data, struct('layout', layout, 'auto_open', false, 'fileopt', 'overwrite', 'filename', 'overlaid-histogram'));
plot_url = response.url